src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonRow } from '../PersonRow';
+import { Params } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Try to rewrite the code so that it is not necessary to disable the linter

Reply:
i rid that